pallet-chainlink-feed/src/lib.rs

```diff
@@ -1374,7 +1397,7 @@ pub mod pallet {
 				Error::<T>::NotPendingPalletAdmin
 			);
 
-			PendingPalletAdmin::<T>::take();
+			PendingPalletAdmin::<T>::kill();
```

Comment:
Is this related to this feature, or it was a previous error? Might affect the rest of the admin transfer logic?

Reply:
`kill` is the same as `take` but does not return the value, I spotted this when I added the new extrinsic,